WhosAGoodBoy/Who's A Good Boy/Assets/Scripts/Interaction/Interactables/House.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class House : Interactable
```

Comment:
Does C# / Unity have a concept of an interface / trait? I don't like hiding that House : MonoBehaivor but maybe that's standard. Also, what if House needs to implement other behavior? Does C# have multiple inheritance?

Reply:
c# doesn't support multiple inheritance. It supports interfaces though.

Reply:
Does an Interactable interface work?